roles/ha-shared-store/README.md

```diff
@@ -9,9 +9,26 @@ Requirements
 This role depends on cluster role provided in this git repository and builds upon it
 High Availability mode with shared-store replication.
 
+Variables
+------------
+
+| Name              | Default Value       | Description          |
+|-------------------|---------------------|----------------------|
+|amq_broker_cluster_broadcast_group_name | my-broadcast-group | name of broadcasting group |
+|amq_broker_cluster_broadcast_group_address | 231.7.7.7 | broadcast IP |
+|amq_broker_cluster_broadcast_group_port | 9876 | broadcasting prot |
```

Comment:
Nope, typo fix commit came after my review. ;)
But thank you, this pull request is all new to me.